### DISCUSSION TITLE
《2022 中国开源发展蓝皮书》数据支撑

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-digger#779] [General] [answered]
去年 OpenDigger 的部分内容支持了《2021 中国开源发展蓝皮书》：http://www.copu.org.cn/new/308

今年我们继续，和 Frank 讨论后，今年可以直接提供一个 Jupyter Notebook 的文件，然后让编委方进行加工，这样我们给不同方面输出的成果能够标准化，便于我们开发。

还请 Frank 将框架中的几个内容跑出来（目前还无法做到的删掉即可），我增加点文字，即可交付过去了。（直接参考了我们”GitHub 2021 数字洞察报告“的结构内容 ）

**1.  总体情况**

- 全球日志事件、活跃仓库数量、活跃开发者数量，近5年的总体数量与每年的增长率。
- 中国日志事件、活跃仓库数量、活跃开发者数量，近5年的总体数量与每年的增长率。

即这部分，展示全球以及中国的整体趋势状况即可，如果能有些时空分布图则更好，特别是全球开发者的地理位置分布。

**2. 中美企业对比情况**

- 2021年美国企业活跃度 Top10 榜单，包括：排名、企业名、总活跃度、日均活跃度、项目总数、参与贡献者总数，人均活跃度
- 2021年中国企业活跃度 Top10 榜单，包括：排名、企业名、总活跃度、日均活跃度、项目总数、参与贡献者总数，人均活跃度
- 2021年美国企业影响力 Top10 榜单，包括：排名、企业名、影响力、项目总数、参与贡献者总数，人均影响力
- 2021年中国企业影响力 Top10 榜单，包括：排名、企业名、影响力、项目总数、参与贡献者总数，人均影响力
- 近十年中国厂商开源活跃度 TOP 10。（类似《中国十年开源洞察报告》）
- 近十年中国厂商开源影响力 TOP 10。
- 某种方式的对比，最好能体现中国某一个方面的上升或接近（项目数量、参与人数、总体/人均等）

**3. 中国项目情况**

- GitHub 2021 年全球项目活跃度 Top 20
- GitHub 2021 年中国项目活跃度 Top 20
- GitHub 2021 年全球项目影响力 Top 20
- GitHub 2021 年中国项目影响力 Top 20
- 近5年，中国 Top10 项目活跃度变化趋势图
- 近5年，中国 Top10 项目影响力变化趋势图
- 某种方式的对比，最好能体现中国某一个方面的上升或接近（项目活跃度/影响力、参与人数、总体/人均等）

以上内容是我想到的适合放到《2022 中国开源发展蓝皮书》，可以任意修改和删减~ @frank-zsy

**Comment by frank-zsy** [comment, marked as answer]
Done by #783 #784

**Reply by frank-zsy** [reply]
https://github.com/X-lab2017/open-digger/blob/master/notebook/Chinese_open_source_blue_paper_2022.ipynb

**Comment by will-ww** [comment]
蓝皮书已发布，内容基本全部采纳，就是他们不太喜欢标注链接和出处~ :-(

https://gitcode.net/COPU/2022

- [第三章 中国开源项目发展现状](https://gitcode.net/COPU/2022/-/blob/master/%E3%80%8A2022%E4%B8%AD%E5%9B%BD%E5%BC%80%E6%BA%90%E5%8F%91%E5%B1%95%E8%93%9D%E7%9A%AE%E4%B9%A6%E3%80%8BV0.8%E7%89%88/%E7%AC%AC3%E7%AB%A0%20%E4%B8%AD%E5%9B%BD%E5%BC%80%E6%BA%90%E9%A1%B9%E7%9B%AE%E5%8F%91%E5%B1%95%E7%8E%B0%E7%8A%B6.md)

**Reply by yoyo-wu98** [reply]
应该是可以进去提一下 issue 的？
